src/data/set/basic.lean

```diff
@@ -291,6 +291,8 @@ in theorem assumptions instead of `∃ x, x ∈ s` or `s ≠ ∅` as it gives ac
 to the dot notation. -/
 protected def nonempty (s : set α) : Prop := ∃ x, x ∈ s
 
+@[simp] lemma nonempty_iff (s : set α) : nonempty ↥s ↔ s.nonempty := nonempty_subtype
+
```

Comment:
The name `set.nonempty_iff` suggests `s.nonempty` appears on the left side of the iff. How about:
```suggestion
@[simp] lemma coe_nonempty_iff (s : set α) : nonempty ↥s ↔ s.nonempty := nonempty_subtype
```
or
```suggestion
@[simp] lemma nonempty_subtype_iff (s : set α) : nonempty ↥s ↔ s.nonempty := nonempty_subtype
```

Reply:
Both good suggestions.

I’m traveling for the next 11 days with only phone for internet so I’ll have to leave this either to your judgement, or till my return.

Thanks for the review.

Reply:
I like both the names you suggested here @Vierkantor but in the end I went with `nonempty_coe_sort`.